core/tests/protractor_desktop/wipeout.js

```diff
@@ -50,10 +51,12 @@ describe('When account is deleted it', function() {
     await users.createAndLoginUser('user1@delete.com', 'userToDelete1');
     await deleteAccountPage.get();
     await deleteAccountPage.requestAccountDeletion('userToDelete1');
+    await waitFor.pageToFullyLoad();
```

Comment:
This function only works on pages that have full-page loading messages, which `/pending-account-deletion` doesn't appear to have. To see this, try loading https://www.oppia.org/pending-account-deletion

ditto for the rest of this file

Reply:
Without this change I was getting an error something like, `expected http://localhost/redirect_url?=pending-acount-deletion to be http://localhost/pending-acount-deletion` it wasn't waiting for page to resolve the URL

Reply:
My guess is that adding `waitFor.pageToFullyLoad();` works only because it introduces a bit of a delay to let the page load. A more reliable solution would be to wait for some element on that page